packages/text/src/Text.ts

```diff
@@ -559,6 +559,9 @@ export class Text extends Sprite
 
         super.destroy(options);
 
+        // set canvas width and height to 0 to make sure it's GC'd by some versions of Safari
```

Comment:
Do you have any source for that optimization?
If so please add the link with tsdoc or in comment.